src/VisualStudio/VisualBasic/Impl/ProjectSystemShim/ConvertedVisualBasicProjectOptions.vb

```diff
@@ -77,20 +77,32 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ProjectSystemShim
                     kind = OutputKind.WindowsRuntimeMetadata
             End Select
 
+            ' GetSDKPath can return E_NOTIMPL if there is no SDK path at all
+            Dim sdkPath As String = Nothing
+            Dim sdkPathHResult = compilerHost.GetSdkPath(sdkPath)
+
+            If sdkPathHResult = VSConstants.E_NOTIMPL Then
+                sdkPath = Nothing
+            Else
+                Marshal.ThrowExceptionForHR(sdkPathHResult, New IntPtr(-1))
```

Comment:
I don't understand why that PR was rejected, but my suspicion is it wasn't because this pattern is bad per-se, but simply because that other bug didn't meet the bar on it's own since the scenario is fairly obscure.